service/src/test/java/greencity/service/UserServiceImplTest.java

```diff
@@ -77,6 +77,13 @@ void findByIdTest() {
         verify(userRepo, times(1)).findById(id);
     }
 
+    @Test
+    void saveTest() {
+        userService.save(userVO);
+        verify(modelMapper).map(userVO, User.class);
+        verify(userRepo, times(1)).save(any());
```

Comment:
` times(1)` could be removed as in the previous line
 verify w/o the second param means 1 time

Reply:
fixed